hybrasyl/Game.cs

```diff
@@ -368,6 +374,16 @@ public static void Main(string[] args)
 
             _spawnThread = new Thread(_monolith.Start);
 
+            _controlThread = new Thread(_monolithControl.Start);
+
+            //foreach (var control in _monolithControls)
+            //{
+            //    Task.Run(() =>
+            //    {
+            //        control.Start();
+            //    });
+            //}
+

```

Comment:
I'd really rather not leave commented code in, in general, anywhere, in PRs. Either use it or take it out.

Reply:
__

Reply:
just junk it

Reply:
I already did. However, I can find plenty of other places where commented code remains.. if you'd like to start getting technical on this.

Reply:
Yeah, the point is, those need to be cleaned up too, and we should not add more.